api.ts

```diff
@@ -144,25 +144,16 @@ export interface Block {
   view_change_proofs: string[];
 }
 
-export type Transaction =
-  | Tagged<"Committed", CommittedTransaction>
-  | Tagged<"Rejected", RejectedTransaction>;
-
-export interface CommittedTransaction {
-  hash: string
-  /**
-   * WIP zeroed
-   */
+export interface TransactionDTO {
+  hash: string;
   block_hash: string;
   payload: TransactionPayload;
   signatures: Signature[];
-}
-
-export interface RejectedTransaction extends CommittedTransaction {
   /**
-   * List of serialized {@link @iroha2/data-model#TransactionRejectionReason}
+   * If the transactionQueryResults in error, this field will contain the rejection reason as a string.
+   * If not, this field will not be present (undefined or null).
```

Comment:
The typing of `rejection_reason` certainly says that it is `undefined | string`. There will be never a `null` type.

Reply:
Addressed in ```api.ts```